src/index.ts

```diff
@@ -0,0 +1 @@
+import {Configuration} from 'xdb-ts-api'
```

Comment:
`xdb-ts-api` is gonna be added as a dependency?

Reply:
Yeah, `xdb-ts-api` is the generated code of protocol between SDK and server